[2025-02-11T00:15:48.639-0300] {processor.py:186} INFO - Started process (PID=71432) to work on /home/felipe/lab/novamind-flow/.env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T00:15:48.640-0300] {processor.py:914} INFO - Processing file /home/felipe/lab/novamind-flow/.env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-11T00:15:48.641-0300] {logging_mixin.py:190} INFO - [2025-02-11T00:15:48.641-0300] {dagbag.py:588} INFO - Filling up the DagBag from /home/felipe/lab/novamind-flow/.env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T00:15:48.644-0300] {logging_mixin.py:190} INFO - [2025-02-11T00:15:48.644-0300] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/felipe/lab/novamind-flow/.env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-11T00:15:48.644-0300] {logging_mixin.py:190} INFO - [2025-02-11T00:15:48.644-0300] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-11T00:15:48.644-0300] {processor.py:927} WARNING - No viable dags retrieved from /home/felipe/lab/novamind-flow/.env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T00:15:48.652-0300] {processor.py:208} INFO - Processing /home/felipe/lab/novamind-flow/.env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2025-02-11T00:16:57.871-0300] {processor.py:186} INFO - Started process (PID=71595) to work on /home/felipe/lab/novamind-flow/.env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T00:16:57.873-0300] {processor.py:914} INFO - Processing file /home/felipe/lab/novamind-flow/.env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-11T00:16:57.875-0300] {logging_mixin.py:190} INFO - [2025-02-11T00:16:57.875-0300] {dagbag.py:588} INFO - Filling up the DagBag from /home/felipe/lab/novamind-flow/.env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T00:16:57.878-0300] {logging_mixin.py:190} INFO - [2025-02-11T00:16:57.878-0300] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/felipe/lab/novamind-flow/.env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-11T00:16:57.878-0300] {logging_mixin.py:190} INFO - [2025-02-11T00:16:57.878-0300] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-11T00:16:57.879-0300] {processor.py:927} WARNING - No viable dags retrieved from /home/felipe/lab/novamind-flow/.env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-11T00:16:57.886-0300] {processor.py:208} INFO - Processing /home/felipe/lab/novamind-flow/.env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
